extensions/notebook/src/book/bookTreeView.ts

```diff
@@ -49,10 +61,41 @@ export class BookTreeViewProvider implements vscode.TreeDataProvider<BookTreeIte
 		return tableOfContentPaths;
 	}
 
+	async openBook(resource: string, context: vscode.ExtensionContext, openAsUntitled: boolean): Promise<void> {
+		try {
+			this.initialze(null, resource, context);
+			let bookViewer = vscode.window.createTreeView('bookTreeView', { showCollapseAll: true, treeDataProvider: this });
+			vscode.commands.executeCommand('workbench.files.action.focusFilesExplorer').then(res => {
+				this._openAsUntitled = openAsUntitled;
+				let books = this.getBooks();
+				if (books && books.length > 0) {
+					bookViewer.reveal(books[0], { expand: 3, focus: true, select: true });
```

Comment:
both are different properties. expand here accepts either Boolean or a number that indicates the number of levels to expand to. But I suppose there's no harm in using the TreeItemCollapsibleState since it returns an number too.